### PR TITLE
Bulk scanning validation endpoint

### PR DESCRIPTION
**Pull request by jakub-kozlowski**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BSP-70

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/AuthService.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import uk.gov.hmcts.reform.authorisation.validators.AuthTokenValidator;
+import uk.gov.hmcts.reform.finrem.caseorchestration.error.ForbiddenException;
+import uk.gov.hmcts.reform.finrem.caseorchestration.error.UnauthenticatedException;
+
+import java.util.List;
+
+import static org.apache.commons.lang3.StringUtils.isBlank;
+
+@Service
+public class AuthService {
```
Comment (yarpo) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/AuthService.java]: I wander if we couldn't move it to div-common-lib and re-use both in FR and Div?

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/bulkscan/OcrValidationControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        super.setUp();
+    }
+
+    @Test
+    public void should_return_401_status_when_auth_service_throws_unauthenticated_exception() throws Exception {
+        String requestBody = readResource(BULKSCAN_VALIDATION_BASIC_FORM_JSON);
+        given(authService.authenticate("")).willThrow(UnauthenticatedException.class);
+
+        mvc
+                .perform(
```
Comment (yarpo) [inline src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/bulkscan/OcrValidationControllerTest.java]: wow, 8 spaces?

File: src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/bulkscan/OcrValidationController.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.controllers.bulkscan;
```
Comment (iichr) [inline src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/bulkscan/OcrValidationController.java]: 👍 I'll move the transformation controller into `bulkscan` as well

File: src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/bulkscan/OcrValidationControllerTest.java (hunk 2 above)
Comment (jakub-kozlowski) [reply src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/bulkscan/OcrValidationControllerTest.java]: it was IntelliJ auto-formatting - maybe I'm missing some formatting setting, do you have formatting template for IntelliJ in divorce?

I'll fix it in the branch implementing validation rules.

**Pull request merged**